x-pack/plugins/synthetics/public/legacy_uptime/state/api/utils.ts

```diff
@@ -10,6 +10,7 @@ import { formatErrors } from '@kbn/securitysolution-io-ts-utils';
 import { HttpFetchQuery, HttpSetup } from '@kbn/core/public';
 import { FETCH_STATUS, AddInspectorRequest } from '@kbn/observability-shared-plugin/public';
 
+type Params = HttpFetchQuery & { version?: string };
```

Comment:
Happy to have this changed, I just tried to take the fewest steps to passing would-be required version to the HTTP requests.

Reply:
We discussed this in the team sync and other possible edge cases. The safest bet is to require a version and leave it to consumers to decide which version they need.

Reply:
We discussed this in the team sync and other possible edge cases. The safest bet is to require a version and leave it to consumers to decide which version they need.